biofefi/services/configuration.py

```diff
@@ -61,3 +61,23 @@ def save_options(path: Path, options: T):
     options_json = dataclasses.asdict(options)
     with open(path, "w") as json_file:
         json.dump(options_json, json_file, indent=4)
+
+
+def load_fi_options(path: Path) -> FeatureImportanceOptions:
+    """Load feature importance options.
+
+    Args:
+        path (Path): The path to the feature importance options file.
+
+    Returns:
+        FeatureImportanceOptions: The feature importance options.
+    """
+
+    try:
+        with open(path, "r") as file:
+            fi_json_options = json.load(file)
+            fi_options = FeatureImportanceOptions(**fi_json_options)
+    except FileNotFoundError:
+        fi_options = None
```

Comment:
Now that I think of it, can you add an except block for `TypeError` in case the json as the wrong arguments in it. Also can you add one for `Exception` as a catch-all for any other error